apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/toggle-sidebar-button/index.tsx

```diff
@@ -26,6 +27,7 @@ export default function ToggleSidebarButton() {
 
 	return (
 		<Button
+			aria-label={ __( 'Open sidebar', 'full-site-editing' ) }
```

Comment:
Nice, I'm definitely hearing this label in Voice Over now. Up until now we haven't had a user visible name for this sidebar feature, but having this label forces us to choose a name. I feel like "sidebar" is too generic (there's lots of sidebars in the editor). Maybe we should label this button something like "Open navigation sidebar", and then carry the "navigation sidebar" name throughout the rest of the labels.

If you hover over the other buttons on the editor toolbar you'll see that there's actually toolitips. Generally I think all the buttons have tooltips so we should do the same here. I think changing the `aria-label` prop to a `label` prop, and adding the `showTooltip` prop to this button will show a tooltip.

Reply:
The announcement post actually called it the Block Editor Sidebar
https://wordpress.com/blog/2020/08/12/manage-posts-and-pages-with-the-block-editor-sidebar/

Reply:
Hee Hee, to your point 
> (there's lots of sidebars in the editor).


the  subheading in the article  is  **"Wait, another sidebar?"**

😄

Reply:
I like Block Editor Sidebar, and I see you've made that change here https://github.com/Automattic/wp-calypso/pull/45318/files